pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -142,9 +143,14 @@ func (s *volumeStatCalculator) calcAndStoreStats() {
 		if err != nil {
 			// Expected for Volumes that don't support Metrics
 			if !volume.IsNotSupported(err) {
-				klog.V(4).InfoS("Failed to calculate volume metrics", "pod", klog.KObj(s.pod), "podUID", s.pod.UID, "volumeName", name, "err", err)
+				if oldErr, found := errors[name]; !found || oldErr.Error() != err.Error() {
```

Comment:
I don't know details here. WHat would be a typical error? Would the comparison with the old `Error()` will work well? If there are some variable part in the error we will spam logs. maybe a safeguard like a counter of how many times it was reported with the threshold would mitigate this risk here?